test/helpers/application_helper_test.rb

```diff
@@ -87,4 +87,11 @@ def test_generate_link_for
       end
     end
   end
+
+  describe 'link_to exceptions' do
+    test 'test if having a javascript as a path crashes link_to' do
```

Comment:
the test name should say what is the expected result, not the wrong result - e.g. `link_to generates correct html for links to javascript`